src/main/java/racingcar/service/RacingCarGameService.java

```diff
@@ -0,0 +1,43 @@
+package racingcar.service;
+
+import racingcar.model.Car;
+import racingcar.model.RacingCars;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class RacingCarGameService {
+
+    private static final String CAR_NAME_SPLIT_REGEX = ",";
+
+    private RacingCars racingCars;
+
+    private List<Car> cars = new ArrayList<>();
+
+    public void generateRacingCars(String inputCarName) {
+        for (String carName : inputCarName.split(CAR_NAME_SPLIT_REGEX)) {
+            Car generatedCar = new Car(carName);
+            cars.add(generatedCar);
```

Comment:
```suggestion
            this.cars = Arrays.stream(inputCarName.split(CAR_NAME_SPLIT_REGEX))
                    .map(Car::new)
                    .collect(Collectors.toList())
```
stream을 적용하는 것은 어떨까요?

Reply:
오 확실히 더 깔끔해지는 것 같네요! 추천 감사합니다 😃